index.html

```diff
@@ -0,0 +1,151 @@
+ <!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet" />
+	<link href="style.css" type="text/css" rel="stylesheet" />
+	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+		<title>Chris Walton</title>
+</head>
+
+<body>
+	<section class="nav-container">
+		<div class="nav-links">
+			<a href="#about-me">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact-me">Contact</a>
+		</div>
+		<div class="nav-cv">
+			<a href="crwaltoncv.docx">Download CV</a>
+		</div>
+		<div class="nav-drop">
+			<button class="dropbtn">Menu</button>
+			<div class="dropdown-content">
+				<a href="#about-me">About</a>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#contact-me">Contact</a>
+				<a href="#crwaltoncv.docx">Download CV</a>
+			</div>
+		</div>	
+	</section>
+	
+	<header>
+		<h1>Chris Walton</h1>
+	</header>
+	
+	<section class="about-container" id="about-me">
+		<div class="about-writing">
+			<div class="about-align">
+				<h3>ABOUT ME</h3>
+				<br>
+				<p>I am a software developer based in Cardiff, with an interest in "INSERT AS APPROPRIATE". I will graduate from iO Academy in October 2022.</p>
+				<br>
+				<p>Prior to my training, I was completing a biology degree and have a deep interest in conservation and the environment. My strengths include communication skills and problem-solving.</p>
+				<br>
+				<p>Outside of work I enjoy reading science fiction novels, playing Dungeons & Dragons with a group of close friends, hiking around South Wales and watching rugby. I also have an minor obsession with cooking and love discovering new restaurants.</p>
```

Comment:
Rather than using br tags here it would be better to use margins or padding - try and keep all visual stuff (including spacing) in the world of CSS